src/main/java/net/fabricmc/loader/launch/FabricTweaker.java

```diff
@@ -59,7 +59,8 @@ public String getTargetNamespace() {
 
 	@Override
 	public void acceptOptions(List<String> localArgs, File gameDir, File assetsDir, String profile) {
-		arguments.parse(localArgs);
+		arguments = localArgs.toArray(new String[0]);
+		/*arguments.parse(localArgs);
```

Comment:
I guess this needs to come back?

Is there a need to support launchwrapper? Could possibly remove its support?

Reply:
It should be redundant since launchwrapper determines the values by parsing "localArgs", but I'm not 100% sure if another desirable environment works differently (thus only commented out). The changed code needs the original args to unify parsing with the Knot path (+prep for game separation).

Launchwrapper is a supported environment, not sure about removing it. This PR isn't the right place either way IMO.

Reply:
Yeah, dont remove it in this PR, thanks for checking though.